relnotes/order.feature.md

```diff
@@ -0,0 +1,7 @@
+### Changed order of app extensions
+
+Previously several extensions used in DaemonApp had same order as application
```

Comment:
Maybe put this description in the commit message too?

Reply:
Sure, done